### ISSUE TITLE
Call a function "while or before" IsoTope is in animation-state?

### ISSUE THREAD

**Issue by nemeth-it** [metafizzy/isotope#189]
Hello,

I'm trying to figure out, how to call a function before or while IsoTope animation running. (reordering)

I found out how i can send a callback after a reLayout. (thanks: onLayout) But i want to send a animation variable (as in-animation trigger), so that my another function doesn't disturbing IsoTope while it's reordering. (in Animation state)

The background of this is a little performance improvement. The CSS3 "box-shadow" attribute can decrease the performance (FPS) of the browsers, while something moving or animating. (like IsoTope) So I fire a resize-end function (not more as a delay). But i want to check if IsoTope is already finished with his animation.

To put it simply: I want to send a variable while animation process that says "Hello, the animations-state is not finished yet!" and the delay function knows then to restart the delay and ask then again.)

I only need to know if it's possible to call a function or variable while IsoTope reorders/animate.
